packages/ts-docs-gen/src/generator-helpers.ts

```diff
@@ -298,21 +318,29 @@ export namespace GeneratorHelpers {
         return `${name}(${parametersString})`;
     }
 
-    export function GetApiItemsFromReferenceTuple<T extends Contracts.ApiItemDto>(
-        items: Contracts.ApiItemReferenceTuple,
+    export function GetApiItemsFromReference<T extends Contracts.ApiItemDto>(
+        items: Contracts.ApiItemReference[],
         extractedData: ExtractDto
     ): T[] {
         const apiItems: T[] = [];
 
         for (const itemReferences of items) {
-            const [, references] = itemReferences;
-
-            for (const reference of references) {
-                const apiItem = extractedData.Registry[reference] as T;
+            for (const referenceId of itemReferences.Ids) {
+                const apiItem = extractedData.Registry[referenceId] as T;
                 apiItems.push(apiItem);
             }
         }
 
         return apiItems;
     }
+
+    export function LogWithApiItemPosition(logLevel: LogLevel, apiItem: Contracts.ApiItemDto, message: string): void {

```

Comment:
Maybe default value for `logLevel` would be handy?

```typescript
export function LogWithApiItemPosition(apiItem: Contracts.ApiItemDto, message: string, logLevel: LogLevel = LogLevel.Warning): void
```
(default value can be other than `LogLevel.Warning`)

Reply:
I am not really sure if it's right decision. Because default Log function for logging is:
```typescript
Log: (level: LogLevel, ...messages: any[]) => number;
```